src/utils/constants.ts

```diff
@@ -1,13 +1,10 @@
-const baseAPIURL: string = import.meta.env.NB_API_QUERY_URL;
-export const queryURL: string = baseAPIURL.endsWith('/')
-  ? `${baseAPIURL}query?`
-  : `${baseAPIURL}/query?`;
-export const attributesURL: string = baseAPIURL.endsWith('/')
-  ? `${baseAPIURL}attributes/`
-  : `${baseAPIURL}/attributes/`;
-export const nodesURL: string = baseAPIURL.endsWith('/')
-  ? `${baseAPIURL}nodes`
-  : `${baseAPIURL}/nodes`;
+export const baseAPIURL: string = import.meta.env.NB_API_QUERY_URL.endsWith('/')
+  ? import.meta.env.NB_API_QUERY_URL
+  : `${import.meta.env.NB_API_QUERY_URL}/`;
```

Comment:
I think it's still cleaner here to read the ENV variable just once, into a local variable,
and then do the slash handling in the way you did before. 
```suggestion
const userAPIURL: string = import.meta.env.NB_API_QUERY_URL
export const baseAPIURL: string = userAPIURL.endsWith('/')
  ? userAPIURL
  : `${userAPIURL}/`;
```
or something like that